tests/unit_and_functional/test_cli.py

```diff
@@ -231,34 +230,24 @@ def test_deployment_test_locally_option_calls_run_workflow_handler(
 
 def test_cli_deployment_handler_error_handling():
     process_one = run(
-        ["bodywork", "deployment", "create", "--namespace=bodywork-dev"],
+        ["bodywork", "deployment", "logs"],
         encoding="utf-8",
         capture_output=True,
     )
-    assert "please specify --name for the deployment" in process_one.stdout
+    assert "please specify --name for the job" in process_one.stdout
```

Comment:
It should still be `deployment` or `deployment-job`? Otherwise it might get confusing.

Reply:
We're mixing terminology too much from between deployment, workflow, job etc. I think deployment job sounds the clearest to me.

Reply:
I take that back, let's keep it deployment. We keep it deployment to the user and it's just us who will refer to it as a job.